src/evm/contract_utils.rs

```diff
@@ -323,8 +324,22 @@ impl ContractLoader {
     ) -> Self {
         let contract_name = prefix.split('/').last().unwrap().replace('*', "");
 
+        let contract_balance = if !constructor_args.is_empty() {
```

Comment:
Don't make value a forced thing for backward compatibility

Reply:
fix it